test/registration.js

```diff
@@ -159,7 +159,6 @@ test.serial('refresh individual instance', async t => {
 		'ncg-graphic-instance:last-of-type',
 		'#reloadButton',
 	);
-	await reload.hover();
```

Comment:
Can you explain why this was removed?

Reply:
This one I'm less sure about than the other one. This line occasionally caused failures with `registration » refresh individual instance` with the message `Node is either not visible or not an HTMLElement`. It didn't seem instrumental to the test (the goal is to click the element, it looked like hover was there because it was trying to simulate a human?), and the other instance where the reload button was clicked did not have the hover.

Basically removed for consistency reasons and because of the false negative I received when it existed.

Reply:
Kk, that sounds reasonable. I admit I have no idea why it was there to begin with.

Reply:
I think I added the line because sometimes I couldn't click the desired element and some stackoverflow suggested hovering before click, if I remember correctly.
If the tests work fine now it should be OK.